DOCS/glauth-tutorial.md

```diff
@@ -0,0 +1,123 @@
+This tutorial aims to provide a general walkthrough to set up a fully working
+GLAuth server using `glauth-k8s` charmed operator, MicroK8s, and Juju.
+
+# Set up the environment
+
+Follow
+this [guide](https://juju.is/docs/juju/set-up--tear-down-your-test-environment)
+to bootstrap a MicroK8s cloud running a Juju controller.
+
+Create a Juju model:
+
+```shell
+juju add-model dev
```

Comment:
Followed [this Markdown rule](https://github.com/DavidAnson/markdownlint/blob/v0.34.0/doc/md014.md) to omit the `$` sign.